code/modules/overmap/sectors.dm

```diff
@@ -58,6 +61,13 @@
 	if(. == INITIALIZE_HINT_QDEL)
 		return
 
+	if(sector_flags & OVERMAP_SECTOR_KNOWN)
+		LAZYADD(global.known_overmap_sectors, src)
+		layer = ABOVE_LIGHTING_LAYER
```

Comment:
these are probably meaningless since sensor stuff isn't using lighting anymore, layer and plane i mean